### DISCUSSION TITLE
Mail / Server without function

### DISCUSSION THREAD

**Discussion by manfahrer** [nextcloud/all-in-one#1449] [Questions] [answered]
### Steps to reproduce
1. open cloud
2. email server settings

### Expected behavior 
After setting, a mail should be sent

### Actual behavior 
Mail without function. both SMPT or sendmail

### Host OS 
Ubuntu 22.04
Hetzner Cloud Server
#### Nextcloud AIO Version 
3.00

#### Current channel 
latest (27.11.2022) and beta (27.11.2022)
new Installation (27.11.2022) 

#### More valuable information 
<html>
<body>

LOG:
Error: fwrite(): Write of 14587 bytes failed with errno=32 Broken pipe at /var/www/html/3rdparty/swiftmailer/swiftmailer/lib/classes/Swift/Transport/StreamBuffer.php#223

sendmail: Beim Senden der E-Mail ist ein Problem aufgetreten. Bitte überprüfen Sie Ihre Einstellungen. (Fehler: Expected response code 220 but got an empty response)

SMTP: Beim Senden der E-Mail ist ein Problem aufgetreten. Bitte überprüfen Sie Ihre Einstellungen. (Fehler: Expected response code 354 but got code "554", with message "554 5.5.1 Error: no valid recipients ")

--

**Comment by szaimen** [comment]
Yeah you need to configure it yourself in the admin settings

**Comment by manfahrer** [comment]
[nextcloud.log](https://github.com/nextcloud/all-in-one/files/10098083/nextcloud.log)
I have done both SMPT and sendmail does not work 

![Bild_2022-11-27_115520656](https://user-images.githubusercontent.com/5245638/204131554-77659108-9e5d-4163-97b7-f42779c4595c.png)

**Comment by manfahrer** [comment, marked as answer]
I think I have found the solution: Why can't I send mail from my server? Unfortunately, the targeted sending of spam mails is a major problem with cloud providers. To counteract this as much as possible, we at Hetzner block outgoing traffic on ports 25 and 465 on all cloud servers by default.
